src/mutation.cpp

```diff
@@ -891,6 +895,12 @@ void player::remove_mutation(std::string mut)
                 break;
             }
         }
+        for( size_t i = 0; i < mutation_data[traits_iter.first].prereqs2.size(); i++ ) {
+            if( mutation_data[traits_iter.first].prereqs2[i] == traits_iter.first ) {
+                dependant.push_back( traits_iter.first );
+                break;
+            }
+        }
```

Comment:
Instead of using `mutation_data[traits_iter.first]`, maybe use `mutation_data[mut]`. That would check for the prerequisite of the mutation that is going to be removed.